spec/services/irs_attempts_api/attempt_event_spec.rb

```diff
@@ -32,7 +32,13 @@
     it 'returns a JWE for the event' do
       jwe = subject.to_jwe
 
+      header_str, *_rest = JWE::Serialization::Compact.decode(jwe)
+      headers = JSON.parse(header_str)
+
+      expect(headers['alg']).to eq('RSA-OAEP')
```

Comment:
this test would have passed before the PR, I'm just adding it as a regression catch now